test/kpod.bats

```diff
@@ -63,6 +63,22 @@ function teardown() {
 	[ "$status" -ne 0 ]
 }
 
+@test "kpod pull from docker with shortname" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull debian
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi docker.io/debian:latest
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod pull from docker with shortname and tag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull debian:6.0.10
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi docker.io/debian:6.0.10
+	[ "$status" -eq 0 ]
+}
+
```

Comment:
Any long name tests?  Or tests against a local repository?  (That last one might be more entertaining).

Reply:
long names are already tested with a pull from the fedora registry